File: core/src/main/java/org/elkd/core/ElkdRuntimeException.java
package org.elkd.core;

public class ElkdRuntimeException extends RuntimeException {
  public ElkdRuntimeException() {
    super();
  }

  public ElkdRuntimeException(final String message) {
    super(message);
  }

  public ElkdRuntimeException(final String message, final Throwable cause) {
    super(message, cause);
  }
}

File: core/src/main/java/org/elkd/core/ElkdRuntimeException.kt
package org.elkd.core

open class ElkdRuntimeException : RuntimeException {
  constructor() : super() {}

  constructor(message: String) : super(message) {}

  constructor(throwable: Throwable) : super(throwable) {}

  constructor(message: String, cause: Throwable) : super(message, cause) {}
}
